hw/bsp/dwm1001-dev/src/hal_bsp.c

```diff
@@ -93,6 +118,47 @@ hal_bsp_get_nvic_priority(int irq_num, uint32_t pri)
     return cfg_pri;
 }
 
+int
+config_lis2dh12_sensor(void)
+{
+#if MYNEWT_VAL(LIS2DH12_ONB)
+    int rc;
+    struct os_dev *dev;
+    struct lis2dh12_cfg cfg;
+
+    dev = (struct os_dev *) os_dev_open("lis2dh12_0", OS_TIMEOUT_NEVER, NULL);
```

Comment:
in mynewt code it is custom not to put space after finishing ) in casts like this
```c
dev = (struct os_dev *)os_dev_open(....
```